ble/services/DFUService.h

```diff
@@ -101,7 +103,15 @@ class DFUService {
                 handoverCallback();
             }
 
-            bootloader_start();
+            // Call bootloader_start implicitly trough a event handler call
+            // it is a work around for bootloader_start not being public in sdk 8.1
+            ble_dfu_t p_dfu;
+            ble_dfu_evt_t p_evt;
+
+            p_dfu.conn_handle = params->connHandle;
+            p_evt.ble_dfu_evt_type = BLE_DFU_START;
+
+            dfu_app_on_dfu_evt(&p_dfu, &p_evt);
```

Comment:
you might as well call it like `dfu_app_on_dfu_evt(NULL /* unused ble_dfu_t */, &p_evt);